system/security/shared/seclib.hpp

```diff
@@ -461,7 +461,7 @@ static const SecFeatureBit SMF_AuthorizeWorkUnitScope_List    = 0x0100000000;
 static const SecFeatureBit SMF_AuthorizeWorkUnitScope_Named   = 0x0200000000;
 static const SecFeatureBit SMF_GetDescription                 = 0x0400000000;
 static const SecFeatureBit SMF_GetPasswordExpirationDays      = 0x0800000000;
-static const SecFeatureBit SMF_CreateUserScopes               = 0x1000000000;
+//static const SecFeatureBit SMF_CreateUserScopes               = 0x1000000000;//feature removed in 9.x
```

Comment:
out of interest, where was this used? (I can't find any references to this const)

Reply:
This is a feature that was added by Tim for SCAPPS to query a security manager for supported features. 
It added an ISecObject interface, which has methods for a caller to query a security manager for supported features (ex checkImplementedFeatures()). So for instance, you could query the HTPASSWD security manager to see if it supports a specific feature (ex scope checking), instead of calling that method and getting an error. Or call the secManager to get all the supported features in a bitmask. Because CreateUserScopes is no longer part of ISecManager, there is no reason to query for that feature